src/components/TransactionFlow/TxReceipt.tsx

```diff
@@ -64,6 +82,13 @@ interface PendingBtnAction {
   text: string;
   action(cb: any): void;
 }
+
+export interface ITxTransferEvent extends IFullTxHistoryValueTransfer {
+  rate: number;
+  toContact?: ExtendedContact;
```

Comment:
Just making sure, these are not persisted. Correct?

Reply:
Nope, these are just used for display